Makefile

```diff
@@ -39,6 +39,14 @@ else
     $(error Unsupported architecture)
 endif
 
+FEATURE ?=
+ifneq ($(FEATURE),)
+ifneq ($(FEATURE),none)
+COMMA:= ,
+ARCH_CFLAGS := $(ARCH_CFLAGS)+$(subst $(COMMA),+,$(FEATURE))
```

Comment:
What if `make FEATURE=crypto,crc` or `make FEATURE=crypto+crc` is run?

Reply:
Now the command should be `make FEATURE=crypto,crc check`. However, the bigger issue this PR is facing is that CI  is not successfully triggered on armv7

Reply:
Would we rollback older version of `run-on-arch-action`? Meanwhile, we should file a new issue there.

Reply:
I think it was an error caused by I set the workflow yaml in wrong way. If not include this PR, the current CI seems to work fine

Reply:
By the way, you should modify `README.md` for the `FEATURE=` information while running `make check`.

Reply:
Information has been added, and the tests work fine now. I made some silly error.